sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/Components15X8E9LSchemasTasksstatePropertiesTasksPropertiesEntityrecognitionpiitasksItemsAllof1.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.textanalytics.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The
+ * Components15X8E9LSchemasTasksstatePropertiesTasksPropertiesEntityrecognitionpiitasksItemsAllof1
+ * model.
+ */
+@Fluent
+public class Components15X8E9LSchemasTasksstatePropertiesTasksPropertiesEntityrecognitionpiitasksItemsAllof1 {
```

Comment:
same here.
Are other langs seeing this too?

Reply:
talked offline with @suhas92, .NET has the same issue. it is from the swagger. I remove these 3 classes right now, since these class are exactly same as TasksStateTasksEntityRecognitionTasksItem, etc.

Reply:
We can add these classes back once it is fixed. Currently, there is not usage of these classes.

Reply:
Make sure the swagger issue is tracked and resolved.

Reply:
Yes. Suhas has it on the issue tracker.